public/js/controllers/extras.js

```diff
@@ -0,0 +1,68 @@
+
+(() => {
+
+    app.register("extras", class extends Stimulus.Controller {
+        static get targets() {
+            return [ "confirmations" ]
+        }
+
+        initialize() {
+            this.startWsRetryTimer()
+        }
+
+        connect() {
+            this.confirmationsTargets.forEach((el,i) => {
+                this.setConfirmationText(el, el.dataset.confirmations)
+            })
+        }
+
+        setConfirmationText(el, confirmations) {
+            if(confirmations > 0) {
```

Comment:
Please use whitespace consistently, eg
`if (confirmations > 0) {`
and 
`} else {`
ala https://crockford.com/javascript/code.html